docs/project/dogfooding.md

```diff
@@ -185,181 +185,181 @@ $ bin\Debug\net6.0\win-x64\publish\App.exe
+[win-x86-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-win-x86.zip
+[win-x86-zip-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-win-x86.zip.sha512
+[win-x86-nethost-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-nethost-win-x86.zip
+[win-x86-symbols-zip-6.0.X]: https://aka.ms/dotnet/6.0/dailydotnet-runtime-symbols-win-x86.zip
+
+[win-arm64-badge-6.0.X]: https://aka.ms/dotnet/6.0/daily/sharedfx_win-arm64_Release_version_badge.svg?no-cache
+[win-arm64-version-6.0.X]: https://aka.ms/dotnet/6.0/daily/runtime-productVersion.txt
+[win-arm64-installer-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-win-arm64.exe
+[win-arm64-installer-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-win-arm64.exe.sha512
+[win-arm64-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-win-arm64.zip
+[win-arm64-zip-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-win-arm64.zip.sha512
+[win-arm64-nethost-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-nethost-win-arm64.zip
+[win-arm64-symbols-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-symbols-win-arm64.zip
+
+[osx-x64-badge-6.0.X]: https://aka.ms/dotnet/6.0/daily/sharedfx_osx-x64_Release_version_badge.svg?no-cache
+[osx-x64-version-6.0.X]: https://aka.ms/dotnet/6.0/daily/runtime-productVersion.txt
+[osx-x64-installer-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-osx-x64.pkg
+[osx-x64-installer-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-osx-x64.pkg.sha512
+[osx-x64-targz-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-osx-x64.tar.gz
+[osx-x64-targz-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-osx-x64.tar.gz.sha512
+[osx-x64-nethost-targz-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-nethost-osx-x64.tar.gz
+[osx-x64-symbols-targz-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-symbols-osx-x64.tar.gz
+
+[osx-arm64-badge-6.0.X]: https://aka.ms/dotnet/6.0/daily/sharedfx_osx-arm64_Release_version_badge.svg?no-cache
+[osx-arm64-version-6.0.X]: https://aka.ms/dotnet/6.0/daily/runtime-productVersion.txt
+[osx-arm64-installer-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-osx-arm64.pkg
+[osx-arm64-installer-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-osx-arm64.pkg.sha512
+[osx-arm64-targz-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-osx-arm64.tar.gz
+[osx-arm64-targz-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-osx-arm64.tar.gz.sha512
+[osx-arm64-nethost-targz-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-nethost-osx-arm64.tar.gz
+[osx-arm64-symbols-targz-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-symbols-osx-arm64.tar.gz
+
+[linux-x64-badge-6.0.X]: https://aka.ms/dotnet/6.0/daily/sharedfx_linux-x64_Release_version_badge.svg?no-cache
+[linux-x64-version-6.0.X]: https://aka.ms/dotnet/6.0/daily/runtime-productVersion.txt
```

Comment:
tried this link at random and it didn't work?
https://aka.ms/dotnet/6.0/daily/runtime-productVersion.txt

Reply:
Let me go check the promotion build. I saw that runtime had promoted, so just assumed these links would be good now.

Reply:
Oh. It looks like the workaround branch that runtime is using doesn't have the link flattening code in it yet. Will change this PR to not flatten, and then once runtime is back on arcade master for promotion, I can come back and update the links again.

Reply:
Let's just wait until the next arcade promotion is in?

Reply:
Sounds good.